chatops4s-discord/src/main/scala/api/DiscordInbound.scala

```diff
@@ -0,0 +1,29 @@
+package api
+
+import api.Server.getClass
+import cats.effect.IO
+import com.typesafe.scalalogging.Logger
+import models.{Button, ButtonInteraction, InboundGateway, InteractionContext}
+
+import scala.collection.concurrent.TrieMap
+
+class DiscordInbound(verbose: Boolean = false) extends InboundGateway {
+  private final val logger = Logger(getClass.getName)
```

Comment:
you dont have to create logger by hand. One utility of scala-logging is that you can write `extends StrictLogging` and thats it.